packages/angular/package.json

```diff
@@ -49,6 +49,7 @@
     "@angular/platform-browser-dynamic": "11.0.0-rc.2",
     "@angular/router": "11.0.0-rc.2",
     "css-vars-ponyfill": "2.3.1",
+    "ramda": "^0.27.0",
```

Comment:
@mathisscott @hippee-lee take a look at this one. I am pretty sure there can be a better fix for this one. Currently, the angular application refers to `@clr/core/icon` and depends on typescript to resolve from where to get the files `dist/` or `packages/core/src`.  It takes the files from `pacakges/core/src` so when the angular build starts it expects the cds-icon files references to packages to point to `packages/angular/node_modules` instead to `packages/core/node_modules` and in angular node_modules there is no `ramda` package installed

Reply:
`ramda` is a dependency for core. think of it like `lodash` or `underscore`. why wouldn't you have it as a dependency? you have `css-vars-ponyfill` listed on the line above it. same sort of thing.

Reply:
When `angular:build` script starts it also builds the `@clr/core/icon` files. When it starts to build the `core` files the build cannot find a `ramda` package and the build fails. I agree this is a dirty fix and that's why I am asking for little assistance to find a better fix. I've been struggling to find a fix to satisfy the build step in GitHub actions for nearly 4 days and this fix is one of the solutions

Reply:
What I aiming to happen now is to let the typescript when it builds the `@clr/angular` and sees import from `@clr/icon` the path of `@clr/core` to be `/dist/core` instead of `packages/core/src/*` like it is declared in [here](https://github.com/vmware/clarity/blob/next/tsconfig.json#L30)

Reply:
All dependencies of Core would also become dependencies of Angular. Typically applications do not need to worry about this because yarn/npm will resolve those automatically for them.

Yarn would typically handle this in apps but in the monorepo we have to use `nohoist` for the angular projects since it does not play well with yarn. Currently projects like `@clr/react` which depend on it will point to the dist directly like this https://github.com/vmware/clarity/blob/next/packages/react/package.json#L20 This gets around all the weird tsconfig/yarn resolution issues usually.

To make yarn happy when installing prior to the build we create a empty directory in the pre-install
https://github.com/vmware/clarity/blob/next/package.json#L85 

During the core build we copy the build into the node_modules to keep it updated without running yarn again. You can see this copy step here https://github.com/vmware/clarity/blob/next/packages/core/package.js#L86

Reply:
I guess I am doing it wrong. I updated the `package.json` of `@clr/angular` to get the `@clr/core` from "./packages/core/dist/core", also updated the `package.js` of `@clr/core` to also copy the files in `node_modules` of `@clr/angular`. Still facing the same issue as previous builds

Reply:
I think I know why, the angular cli projects dont hoist or share dependencies with the rest of the Repo which is why the react apps work but angular does not. Its a tricky problem with the monorepo we havent really solved in a generic way yet. I can dig into this some more after the workshop.

Reply:
Ok, I believe for now the best way to connect core with everything else is to provide `@clr/core` as a dep. I believe also that providing a package manually inside node_modules is just counterproductive. 

What I figure out yesterday when I did the experiment on what @coryrylan suggested is that:
1. Running the build of `@clr/angular`
2. The builder gets to some `@clr/core/icon` import line
3. The builder tries to resolve the imports inside `@clr/core/icons` by looking into `packages/angular/node_modules`
4. Builder fails because the packages that `@clr/core` depends on are no provided in `@clr/angular` node_modules

We have the same issue inside the website and what we did there is a workaround with webpack where we manually provide where to look for certain packages but I believe this solution is also tricky and not easy to maintain.

In the end, when we publish the `@clr/angular` package it expends on peerDependency of `@clr/core` so the user has to install `@clr/core` anyway. Having this in mind I believe having it as `dependency` inside our internal `angular` package is fine for now cause we only use this dependency internally and we can update it at any time without affecting our users.